pkg/controllers/mcs/service_export_controller.go

```diff
@@ -291,7 +292,10 @@ func (c *ServiceExportController) handleServiceExportEvent(serviceExportKey keys
 	_, err := helper.GetObjectFromCache(c.RESTMapper, c.InformerManager, serviceExportKey)
 	if err != nil {
 		if apierrors.IsNotFound(err) {
-			return cleanupWorkWithServiceExportDelete(c.Client, serviceExportKey)
+			return cleanupWorkWithLabelsSelectorDelete(c.Client, serviceExportKey, labels.SelectorFromSet(labels.Set{
```

Comment:
Sorry, I can not get the benefits of moving `labelselector` to input parameters, can you help remind me?

Reply:
Using label selector to delete `work` derived from `serviceImport` or `serviceExport`. The `work` created by `service_export_controller` and `multiclusterservice_controller` have different labels. Therefore, the `labelSelector` is moved to the input parameter in order to reuse the function `cleanupWorkWithLabelsSelectorDelete`.

Reply:
Thanks, I get it. Putting the `cleanupWorkWithLabelsSelectorDelete` function into a separate file will be better.